src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/Types/DynamicContentTypeBuilder.cs

```diff
@@ -28,16 +29,28 @@ public class DynamicContentTypeBuilder : IContentTypeBuilder
             S = localizer;
         }
 
-        public void Build(ISchema schema, FieldType contentQuery, ContentTypeDefinition contentTypeDefinition, ContentItemType contentItemType)
+        public abstract void Build(ISchema schema, FieldType contentQuery, ContentTypeDefinition contentTypeDefinition, ContentItemType contentItemType);
+
+        protected void BuildObjectGraphType(ISchema schema, FieldType contentQuery, ContentTypeDefinition contentTypeDefinition, IContentItemObjectGraphType objectGraphType)
         {
-            var serviceProvider = _httpContextAccessor.HttpContext.RequestServices;
-            var contentFieldProviders = serviceProvider.GetServices<IContentFieldProvider>().ToList();
+            BuildInternal(schema, contentQuery, contentTypeDefinition, objectGraphType);
+        }
+
+        protected void BuildInputObjectGraphType(ISchema schema, FieldType contentQuery, ContentTypeDefinition contentTypeDefinition, IContentItemInputObjectGraphType inputObjectGraphType)
+        {
+            BuildInternal(schema, contentQuery, contentTypeDefinition, inputObjectGraphType);
+        }
 
+        private void BuildInternal(ISchema schema, FieldType contentQuery, ContentTypeDefinition contentTypeDefinition, IComplexGraphType graphType)
```

Comment:
`contentQuery` is never used
```suggestion
        private void BuildInternal(ISchema schema, ContentTypeDefinition contentTypeDefinition, IComplexGraphType graphType)
```